src/software/ai/hl/stp/action/movespin_action_test.cpp

```diff
@@ -8,7 +8,7 @@ TEST(MoveSpinActionTest, getDestination)
 {
     Robot robot = Robot(0, Point(), Vector(), Angle::zero(), AngularVelocity::zero(),
                         Timestamp::fromSeconds(0));
-    MoveSpinAction action = MoveSpinAction(0.05);
+    MoveSpinAction action = MoveSpinAction(0.05, false);
```

Comment:
Why is the order of params different here? The signature has `loop_forever` as the first param. (Does this compile?)

Reply:
you're right I'll change the order. But surprisingly this does compile, i expected the `explicit` constructor to catch this, it sets `loop_forever = true` and `close_to_dest_threshold = 0`